scripts/mapCoverage.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+/**
+ * Because we have a build step in jest, we collect coverage from both
+ * `packages/src/*` and `package/build/*` files during our test run. They are
+ * practically the same files. This script will merge them together
+ * and write the final coverage report.
+ */
```

Comment:
we can run the same file from `src` and `build`

imagine this

```js
// inside `jest-whatever/src/__tests__/index.js`
require('../index.js'); // this will require `jest-whatever/src/index.js`
```

```js
// inside another packages
require('jest-whatever') // this will go through lerna and require `jest-whatever/build/index.js
```

these files are kind of identical, but the coverage and actual files are actually different